MobileWallet/Screens/Debug/HomeViewControllerDebugExtension.swift

```diff
@@ -103,13 +136,17 @@ extension HomeViewController {
             self.showTariLibLogs()
         }))
 
-        alert.addAction(UIAlertAction(title: "Generate test data", style: .default, handler: { (_)in
-            self.generateTestData()
+        alert.addAction(UIAlertAction(title: "Add custom base node", style: .default, handler: { (_)in
+            self.showAddCustomBaseNode()
         }))
 
-        alert.addAction(UIAlertAction(title: "Simulate recieve transaction", style: .default, handler: { (_)in
-            self.simulateReceieveTransactions()
-        }))
+//        alert.addAction(UIAlertAction(title: "Generate test data", style: .default, handler: { (_)in
```

Comment:
maybe remove this from code?

Reply:
I might want to keep it there. I think maybe it might be needed if you want to work on the UI without running tor and connecting to a base node.